R-package/cran-comments.md

```diff
@@ -1,9 +1,31 @@
 # CRAN Submission History
 
+## v3.3.4 - Submission 1 - (December 15, 2022)
+
+### CRAN response
+
+Accepted to CRAN
+
+### Maintainer Notes
+
+Submitted with the following comment:
+
+> This submission contains {lightgbm} 3.3.3.
```

Comment:
Shouldn't this be 3.3.4?

Reply:
ugh yes you're right, sorry

Reply:
Sorry for the late review

Reply:
ha no problem at all! You are not on a 24/7 on-call rotation here 😂